data/ui/SymbolsView.ui

```diff
@@ -28,21 +28,14 @@
                 <property name="valign">fill</property>
                 <property name="orientation">vertical</property>
                 <child>
-                  <object class="GtkFrame">
+                  <object class="GtkListBox" id="symbols">
```

Comment:
Since you are already using libhandy you could try the premade styleclass maybe.

```
            <style>
              <class name="content"/>
            </style>
```

Reply:
That's a good catch, thanks! Done:

https://github.com/bitstower/markets/commit/ae175fe7be9624caf09da7d6c19951e32a70483f